lib/common/strings.c

```diff
@@ -847,6 +847,44 @@ pcmk__str_none_of(const char *s, ...)
     return rc;
 }
 
+/*!
+ * \internal
+ * \brief Check whether a character is in any of a list of strings
+ *
+ * \param[in]   ch      Character (ASCII) to search for
+ * \param[in]   ...     Strings to search. Final argument must be NULL.
+ *
+ * \return  \c true if any of \p ... contain \p ch, \c false otherwise
+ *
+ * \note    \p ... must contain at least one argument (NULL).
+ */
+bool
+pcmk__char_in_any_str(int ch, ...)
+{
+    bool rc = false;
+    va_list ap;
+
+    /*
+     * Passing a char to va_start() can generate compiler warnings,
+     * so ch is declared as an int.
+     */
+    va_start(ap, ch);
```

Comment:
I think this is fine, since strchr()'s prototype accepts an int already and converts it to a char. I had to make this change after Clang refused to compile.